clustered/integration-test/src/test/java/org/ehcache/clustered/management/ClusteringManagementServiceTest.java

```diff
@@ -395,6 +399,60 @@ public static void initDescriptors() throws ClassNotFoundException {
     CACHE_DESCRIPTORS.add(new StatisticDescriptor("Cache:ClearCount", StatisticType.COUNTER_HISTORY));
     CACHE_DESCRIPTORS.add(new StatisticDescriptor("Cache:MissCount", StatisticType.COUNTER_HISTORY));
     CACHE_DESCRIPTORS.add(new StatisticDescriptor("Cache:MissRatio", StatisticType.RATIO_HISTORY));
+    //TODO CACHE_DESCRIPTORS above will be deletetd once the following task is completed: https://github.com/ehcache/ehcache3/issues/1684
+    /////////////////////////////////////////////////////////////////////////////////////////////////////
+
+    /////////////////////////////////////////////////////////////////////////////////////////////////////
+    //These descriptors are for the new statistis architecture.  These will remain.
+    //task https://github.com/ehcache/ehcache3/issues/1684
+    CACHE_DESCRIPTORS.add(new StatisticDescriptor("Cache:Get:HitNoLoaderCount", StatisticType.COUNTER_HISTORY));
+    CACHE_DESCRIPTORS.add(new StatisticDescriptor("Cache:Get:HitWithLoaderCount", StatisticType.COUNTER_HISTORY));
```

Comment:
I believe the granularity to be too fine here.

Cache:Hit is aggregation of
Cache:Hit:Get -> HitNoLoader + HitWithLoader
Cache:Hit:GetAll -> HitNoLoader + HitWithLoader
Cache:Hit:PutIfAbsentHit -> Hit
Cache:Hit:Replace2ArgHit -> Hit
Cache:Hit:Replace3ArgHit -> Success, Failure_Present
Cache:Hit:Remove2ArgHit -> Success, Failure_Present

Cache:Miss is aggregation of
Cache:Miss:Get -> MissNoLoader + MissWithLoader
Cache:Miss:GetAll -> MissNoLoader + MissWithLoader
Cache:Miss:PutIfAbsentMiss -> Put
Cache:Miss:Replace2ArgMiss -> Miss_Not_Present
Cache:Hit:Replace3ArgMiss -> Miss_Not_Present
Cache:Hit:Remove2ArgMiss -> Failure_Missing

as an example.

This leaves one issue I can see in the naming because some outcomes are shared by high level stats.